net/http/inc/THttpWSEngine.h

```diff
@@ -19,7 +19,7 @@
 class THttpWSEngine {
 
 protected:
-   THttpWSEngine(std::shared_ptr<THttpCallArg> arg);
+   THttpWSEngine() = default;
```

Comment:
If nobody is `#including` this but a ROOT source file then please move this header into net/http/src/ and remove it from Linkdef.h.

Reply:
And here: https://github.com/linev/root/commit/983e97b9b1dd3b3bb50bf12a94ae7aa0c118547f